apps/chat/src/components/Common/NoResultsFound.tsx

```diff
@@ -4,15 +4,30 @@ import { Translation } from '@/src/types/translation';
 
 import Magnifier from '../../../public/images/icons/search-alt.svg';
 
-export const NoResultsFound = () => {
+interface NoResultsFoundProps {
+  iconSize?: number;
+  fontSize?: string;
+  gap?: string;
+}
+
+export const NoResultsFound = ({
+  iconSize = 60,
+  fontSize = 'text-sm',
+  gap = 'gap-3',
+}: NoResultsFoundProps) => {
   const { t } = useTranslation(Translation.Common);
+
   return (
     <div
-      className="flex flex-col items-center justify-center gap-3"
+      className={`flex flex-col items-center justify-center ${gap}`}
```

Comment:
this won't work add `gap` class to `styles` prop if it should be dynamic

Reply:
use `classNames` instead of string interpolation, but I agree with Alexander that it would be better to provide `containerClassName` and `textClassName` instead of specific `gap` and `fontSize`